### PR TITLE
Update SDs for G12 - part 3

### PR DESCRIPTION
**Pull request by suehighmoor**
DO NOT MERGE until G12 goes live

Even more updates to service definitions for G12. Covers:
Dedicated Compute
DRaaS
HPC
Migration to the Cloud
Private Cloud
Private Cloud for Storage
Desktop as a Service

### CODE REVIEW

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

File: articles/daas/daas-sd.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+## What is UKCloud Desktop as a Service?
+
+UKCloud Desktop as a Service provides a platform to deliver, protect and manage desktops and apps whilst containing costs and ensuring that end users can work anytime, anywhere, across any device.
+
+UKCloud Desktop as a Service provides significant security and compliance benefits to your organisation, whilst increasing workforce enablement through smarter more flexible working. This aligns with government policy around smart working within new Government Hubs. UKCloud Desktop as a Service also enables the Bring Your Own Device (BYOD) policy and supports remote working, without compromising application availability, security, and compliance.
+
+## What the service can help you achieve
+
+- **Deliver both persistent and non-persistent Windows 10 desktops.** Supports both standard and complex requirements within your organisation.
```
Comment (gmartinukcloudcom) [inline articles/daas/daas-sd.md:28]: can you please change to read 'Deliver both dedicated and shared Windows 10 desktops" ... this is then consistent with how we describe options later in the document,

**Review by gmartinukcloudcom — CHANGES_REQUESTED** [review]
just one minor wording change on the DaaS text - i take it we can do this ?

File: articles/daas/daas-sd.md (hunk 1 above)
Comment (suehighmoor) [reply articles/daas/daas-sd.md]: @gmartinukcloudcom, made that change if you could just take a look and approve please!

**Review by gmartinukcloudcom — APPROVED** [review]

**Pull request merged**